tests/test_theme.py

```diff
@@ -26,3 +110,265 @@ def test_font():
+
+
+def test_volume_mapper(default_theme):
+    assert hasattr(default_theme, 'volume_mapper')
+    volume_mapper = 'gpu'
+    default_theme.volume_mapper = volume_mapper
+    assert default_theme.volume_mapper == volume_mapper
+
+    with pytest.raises(ValueError, match='unknown'):
+        default_theme.volume_mapper = 'invalid'
+
+
+
+@pytest.mark.parametrize('parm', [('background', (0.1, 0.2, 0.3)),
+                                  ('auto_close', False),
+                                  ('notebook', False),
+                                  ('full_screen', True),
+                                  ('nan_color', (0.5, 0.5, 0.5)),
+                                  ('edge_color', (1.0, 0.0, 0.0)),
+                                  ('outline_color', (1.0, 0.0, 0.0)),
+                                  ('floor_color', (1.0, 0.0, 0.0)),
+                                  ('show_scalar_bar', False),
+                                  ('lighting', False),
+                                  ('interactive', False),
+                                  ('render_points_as_spheres', True),
+                                  ('transparent_background', True),
+                                  ('title', 'test_title'),
+                                  ('multi_samples', 10),
+                                  ('multi_rendering_splitting_position', 0.1),
+                                  ('smooth_shading', True),
+                                  ('name', 'test_theme'),
+])
+def test_theme_parm(default_theme, parm):
+    attr, value = parm
+    assert hasattr(default_theme, attr)
+    setattr(default_theme, attr, value)
+    assert getattr(default_theme, attr) == value
+
+
+def test_theme_colorbar_orientation(default_theme):
+    orient = 'vertical'
+    default_theme.colorbar_orientation = orient
+    assert default_theme.colorbar_orientation == orient
+
+    with pytest.raises(ValueError):
+        default_theme.colorbar_orientation = 'invalid'
+
+
+def test_restore_defaults(default_theme):
+    orig_value = default_theme.show_edges
+    default_theme.show_edges = not orig_value
+    default_theme.restore_defaults()
+    assert default_theme.show_edges == orig_value
+
+
+def test_repr(default_theme):
+    rep = str(default_theme)
+    assert 'Background' in rep
+    assert default_theme.cmap in rep
+    assert str(default_theme.colorbar_orientation) in rep
+    assert default_theme._name.capitalize() in rep
+
+
+def test_theme_slots(default_theme):
+    # verify we can't create an arbitrary attribute
+    with pytest.raises(AttributeError, match='has no attribute'):
+        default_theme.new_attr = 1
+
+
+def test_theme_eq():
+    defa_theme0 = pyvista.themes.DefaultTheme()
+    defa_theme1 = pyvista.themes.DefaultTheme()
+    assert defa_theme0 == defa_theme1
+    dark_theme = pyvista.themes.DarkTheme()
+    assert defa_theme0 != dark_theme
+
+    # for coverage
+    assert defa_theme0 != 'apple', 'wrong type test failed'
```

Comment:
We almost nowhere else have such comments in test assertions, so I'm thinking this one might be a mistake.

Reply:
Removed for consistency.